meta/documents/user_guide/de/initial_setup.md

```diff
@@ -79,36 +79,37 @@ Dieses Verfahren ist nur erforderlich, wenn [die Einstellung `Vollständigen Bes
 
 1. Gehen Sie auf der Hauptseite von Plentymarkets zu `Artikel` >> `Artikel bearbeiten`.
 
-2. Suchen Sie nach Artikeln und öffnen Sie sie
+2.  Suchen Sie nach Artikeln und öffnen Sie sie **`Stellen Sie sicher, dass auf dem Artikel die **`Hauptvariante`** steht`** siehe Bild unten

```

Comment:
This has the same problem as the English version did - the part starting with the bolded text is hanging out at the end of this step without any punctuations.